test/Infrastructure.IntegrationTest/AdminConsole/Repositories/OrganizationUserRepositoryTests.cs

```diff
@@ -354,4 +355,75 @@ await organizationUserRepository.CreateAsync(new OrganizationUser
         Assert.Single(responseModel);
         Assert.Equal(orgUser1.Id, responseModel.Single().Id);
     }
+
+    [DatabaseTheory, DatabaseData]
+    public async Task CreateManyAsync_NoId_Works(IOrganizationRepository organizationRepository,
+        IUserRepository userRepository,
+        IOrganizationUserRepository organizationUserRepository)
+    {
+        // Arrange
+        var user1 = await userRepository.CreateTestUserAsync("user1");
+        var user2 = await userRepository.CreateTestUserAsync("user2");
+        var user3 = await userRepository.CreateTestUserAsync("user3");
+        List<User> users = [user1, user2, user3];
+
+        var org = await organizationRepository.CreateAsync(new Organization
+        {
+            Name = $"test-{Guid.NewGuid()}",
+            BillingEmail = "billing@example.com", // TODO: EF does not enforce this being NOT NULL
+            Plan = "Test", // TODO: EF does not enforce this being NOT NULl
+        });
+
+        var orgUsers = users.Select(u => new OrganizationUser
+        {
+            OrganizationId = org.Id,
+            UserId = u.Id,
+            Status = OrganizationUserStatusType.Confirmed,
+            Type = OrganizationUserType.Owner
+        });
+
+        // This fails because the IDs clash, even though the repository should set these to random values
```

Comment:
Please delete this comment given that the test no longer fails - this was just a signpost for you.